projects/packages/jetpack-mu-wpcom/src/class-jetpack-mu-wpcom.php

```diff
@@ -97,6 +99,11 @@ public static function load_features() {
 		if ( class_exists( 'Automattic\Jetpack\Scheduled_Updates' ) ) {
 			Scheduled_Updates::init();
 		}
+
+		// Gets autoloaded from the Scheduled_Updates package.
+		if ( class_exists( 'Automattic\Jetpack\Calypsoify\Jetpack_Calypsoify' ) ) {
```

Comment:
This works the same but IMO is more readable.
```suggestion
		if ( class_exists( Jetpack_Calypsoify::class ) ) {
```
Although... when wouldn't the class exist, since the package is being required in composer.json?

Reply:
Good point, I just followed the previous process, but then I see the one above does not check for the class `\Marketplace_Products_Updater::init();`. Will change that.